lib/models/label.dart

```diff
@@ -39,4 +40,6 @@ class Label {
         'updated': updated?.toUtc()?.toIso8601String(),
         'created': created?.toUtc()?.toIso8601String(),
       };
+
+  Color get textColor => color.computeLuminance() > 0.5 ? vLabelDark : vLabelLight;
```

Comment:
I needed to add a null check:
`  Color get textColor => color != null && color.computeLuminance() > 0.5 ? vLabelDark : vLabelLight;`
Otherwise, I get a 'computeLuminance was called on null' exception

Reply:
What dart version are you using? Could have something to do with null safety checks.

Reply:
Could be, but I don't think so because if I change to a minimum dart version with sound null safety (eg 2.12.0) the app isn't buildable. To upgrade to null safety will be a big undertaking.

Reply:
It works on my machine<sup>TM</sup> but 🤷. I am using dart version 2.17.6 for reference. Regardless, I added the null check for now anyway. At some point I do want to do the sound null safety upgrade. It is possible to do it in parts so it could be done over time. The Flutter team has a good [video](https://youtu.be/bvq7wbn4VAA) on null safety migration.